process/smartContract/process.go

```diff
@@ -785,11 +779,17 @@ func (sc *scProcessor) ExecuteBuiltInFunction(
 	}
 
 	var vmOutput *vmcommon.VMOutput
-	vmOutput, err = sc.resolveBuiltInFunctions(acntSnd, acntDst, vmInput)
+	vmOutput, err = sc.resolveBuiltInFunctions(vmInput)
 	if err != nil {
 		log.Debug("processed built in functions error", "error", err.Error())
 		return 0, err
 	}
+
+	acntSnd, err = sc.reloadLocalAccount(acntSnd)
```

Comment:
we need to reload here as to reflect the new changes in the acntSnd done in the prepareExecution, after the processSCPayment call

Reply:
the reload is needed to reflect the changes coming from the built in functions call.

Reply:
No, as that function no longer calls the SaveAccount method